package.json

```diff
@@ -47,6 +47,7 @@
     "@dnd-kit/sortable": "^8.0.0",
     "@dnd-kit/utilities": "^3.2.2",
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.3",
+    "@edx/browserslist-config": "1.2.0",
```

Comment:
`browserslist-config` is required for webpack to build the MFE so it should be a dependency, not a devDependency.


Ref: [OEP-67 - Frontend Technology Selection - 15 Development dependencies should be separated into devDependencies](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0067-bp-tools-and-technology.html#frontend-technology-selection)